mesonbuild/compilers/mixins/elbrus.py

```diff
@@ -70,3 +70,14 @@ def get_default_include_dirs(self) -> T.List[str]:
             if line.lstrip().startswith('--sys_include'):
                 includes.append(re.sub(r'\s*\\$', '', re.sub(r'^\s*--sys_include\s*', '', line)))
         return includes
+
+    def get_optimization_args(self, optimization_level: str) -> T.List[str]:
+        return gnu_optimization_args[optimization_level]
+
+    def get_pch_suffix(self) -> str:
+        # Actually it's not supported for now, but probably will be supported in future
+        return 'pch'
+
+    def openmp_flags(self) -> T.List[str]:
+        return ['-fopenmp']
+
```

Comment:
__[Flake8]__ blank line at end of file ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6455#issue-5055562))

<details>

| Rule |
| - |
| W391 |

</details>
<sub>posted by Sider</sub>